opennms-container/horizon/Dockerfile

```diff
@@ -1,19 +1,12 @@
-ARG BASE_IMAGE="opennms/openjdk"
-ARG BASE_IMAGE_VERSION="latest"
+ARG BASE_IMAGE="opennms/base-horizon"
+ARG BASE_IMAGE_VERSION="jdk11-1.0.0"
```

Comment:
The only thing I worry about this is that if you fix a bug in the packaging of the intermediate container, you'll have to change every branch going forward that references this image.  Should there be a `jdk11-1` tag that can be fixed without changing the source?  Semantic versioning style...

Reply:
we provide different tags and less specific floating tags and very specific tags. For example:

* jdk11-1.0.0: which references the last successful build
* jdk11-1.0.0-cb.1035: the exact build produced this image

Floating tags are dangerous cause they make things hard to reproduce. Build a with jdk11-1.0.0 can pass and the next build with a new jdk11-1.0.0 can fail and you don't have a clue why. On the other side if you make a change without touching all references than floating tags help to populate your changes automatically.

We have to make a decision here how we want to proceed here. I've picked the version here with floating tag.

Reply:
Gotcha.  I mentioned semver because it would be a way to attempt to mitigate breaking something on a floating tag, but if it can be overridden anyways, that solves the biggest issue.

Reply:
I'm also very open to version tag ideas. I've just picked 1.0.0 from default semver.org which we can fully control. I'm not sure if we really need 1.0.0 or 1.0 is enough + the additional build number we can't control and comes from the CI/CD to make it absolutely specific.